lib/rules/block-indentation.js

```diff
@@ -387,7 +390,7 @@ export default class BlockIndentation extends Rule {
 
         let warning =
           `Incorrect indentation for \`${displayName}\` beginning at ${startLocation}` +
-          `. Expected \`${display}\` ending at ${endLocation} to be at an indentation of ${expectedEndColumn} but ` +
+          `. Expected \`${display}\` ending at ${endLocation} to be at an indentation of ${expectedEndColumn}, but ` +
```

Comment:
I noticed some inconsistency with the error messages, so I sneaked that in as well :grimacing: